src/app/compare/CompareItem.tsx

```diff
@@ -62,6 +63,17 @@ export default memo(function CompareItem({
         <div className={styles.itemActions}>
           {item.vendor ? (
             <VendorItemWarning item={item} />
+          ) : item.owner !== 'unknown' &&
+            !item.canPullFromPostmaster &&
+            item.location.inPostmaster ? (
```

Comment:
Should this be a function, in case we want to improve it later?

Reply:
Yeah. There's a handful of other places that do this exact same logic so makes sense to pull it until util.